client/pk-console.c

```diff
@@ -1111,6 +1111,16 @@ pk_console_update_packages (PkConsoleCtx *ctx, gchar **packages, GError **error)
 	return TRUE;
 }
 
+static gboolean
+pk_console_update_system_filter_helper(PkPackage *package, gpointer user_data)
+{
+	PkInfoEnum package_enum;
+
+	package_enum = pk_package_get_info(package);
```

Comment:
space after braket, and can probably be moved up to the `PkInfoEnum` line making the entire function only two lines.